database/schema.sql

```diff
@@ -107,36 +111,40 @@ CREATE TABLE public.jobs (
   created_at TIMESTAMP NOT NULL DEFAULT now(),
   updated_at TIMESTAMP NOT NULL DEFAULT now(),
   deleted_at TIMESTAMP,
-  created_by uuid NOT NULL DEFAULT auth.uid()
+  created_by VARCHAR NOT NULL REFERENCES public.users (id)
 );
+COMMENT ON COLUMN public.jobs.created_by IS 'References to a wallet address from users table';
 
 -- table: job_keyword
 CREATE TABLE public.job_keyword (
-  job_id uuid NOT NULL REFERENCES public.jobs (id),
-  keyword_id uuid NOT NULL REFERENCES public.keywords (id),
+  job_id UUID NOT NULL REFERENCES public.jobs (id),
+  keyword_id UUID NOT NULL REFERENCES public.keywords (id),
   created_at TIMESTAMP NOT NULL DEFAULT now(),
   updated_at TIMESTAMP NOT NULL DEFAULT now(),
   deleted_at TIMESTAMP,
+  created_by VARCHAR NOT NULL REFERENCES public.users (id),
   PRIMARY KEY (job_id, keyword_id)
 );
 
 -- table: job_applications
 CREATE TABLE public.job_applications (
-  id uuid PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
-  job_id uuid NOT NULL REFERENCES public.jobs (id),
+  id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  job_id UUID NOT NULL REFERENCES public.jobs (id),
   pitch VARCHAR CHECK (pitch <> ''),
   content TEXT CHECK (content <> ''),
   status VARCHAR CHECK (status <> ''),
   created_at TIMESTAMP NOT NULL DEFAULT now(),
   updated_at TIMESTAMP NOT NULL DEFAULT now(),
   deleted_at TIMESTAMP,
-  created_by uuid NOT NULL DEFAULT auth.uid()
+  created_by VARCHAR NOT NULL REFERENCES public.users (id)
 );
+COMMENT ON COLUMN public.job_applications.created_by IS 'References to a wallet address from users table';
 
 
 -- table: gigs
 CREATE TABLE public.gigs (
-  id uuid PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
Should we keep either a job or a gig? I don't know what the conclusion there was for the MVP

Reply:
I think we're keeping both, the other table could be empty for now when launching MVP